### DISCUSSION TITLE
Allow adding scopes to HasMany fieldtype

### DISCUSSION THREAD

**Discussion by mxaGianluca** [statamic-rad-pack/runway#518] [Feature Requests]
It would be great if you can add a scope to a HasMany relation. This would allow you to configure how the same relation behaves depending on where you use it.

Right now you have the option to add a scope for the `runwayListing` by defining a `scopeRunwayListing` on the model. But i think it would be really handy to be able to add a scope when you configure the relation in the cp.

**Comment by duncanmcclean** [comment]
Statamic's built-in Entries fieldtype has a ["Query Scopes" config option](https://statamic.dev/fieldtypes/entries#options). I'd accept a PR adding the same option to Runway's Belongs To & Has Many fieldtypes.

**Comment by duncanmcclean** [comment]
I've just added a "Query Scopes" option to Runway's relationship fieldtypes, it mirrors how you add query scopes to Statamic's built-in relationship fields.
